types/node/stream/web.d.ts

```diff
@@ -205,7 +205,12 @@ declare module "stream/web" {
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/ReadableStreamBYOBReader) */
     interface ReadableStreamBYOBReader extends ReadableStreamGenericReader {
         /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/ReadableStreamBYOBReader/read) */
-        read<T extends ArrayBufferView>(view: T): Promise<ReadableStreamReadResult<T>>;
+        read<T extends ArrayBufferView>(
+            view: T,
+            options?: {
+                min?: number | undefined;
```

Comment:
Looks redundant to me
```suggestion
                min?: number;
```

Reply:
I believe this is needed.

See https://github.com/DefinitelyTyped/DefinitelyTyped/issues/44317#issuecomment-876540548 and https://www.typescriptlang.org/tsconfig/#exactOptionalPropertyTypes.

Reply:
`streams/web` is an exception – `| undefined` is omitted here to maintain consistency with the TypeScript DOM library, so this change is indeed needed.

Reply:
Ok.. Fixed it.